src/main/java/network/brightspots/rcv/ContestConfig.java

```diff
@@ -222,7 +222,7 @@ private void validateCvrFileSources() {
         }
 
         // perform CDF checks
-        if (source.getProvider().equals("CDF")) {
+        if (source.getProvider() != null && source.getProvider().equals("CDF")) {
```

Comment:
Maybe we should just have an `isCdf` method on CVRSource?

Reply:
Done.